### ISSUE TITLE
cdktf get -l python fails in macosx

### ISSUE THREAD

**Issue by karthikns16** [hashicorp/terraform-cdk#704]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions
cdktf --> 0.3.0
python --> Python 3.8.5
node --> 12.16.1

### Expected Behavior

```cdktf get -l python``` should work

### Actual Behavior

```Error: jsii compilation failed with non-zero exit code: 1
  | Error: Cannot find module 'json-stable-stringify/package.json'
  | Require stack:
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/type-system.js
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/index.js
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/lib/index.js
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/bin/jsii-pacmak.js
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/bin/jsii-pacmak
  |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
  |     at Function.resolve (internal/modules/cjs/helpers.js:83:19)
  |     at TypeSystem._loadModule (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/type-system.js:105:40)
  |     at async TypeSystem._loadModule (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/type-system.js:109:17)
  |     at async TypeSystem.loadModule (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/type-system.js:68:21)
  |     at async /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/lib/index.js:45:13
  |     at async Promise.all (index 0)
  |     at async Object.pacmak (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/lib/index.js:41:5)
  +----------------------------------------------------------------------------------
  | Command: /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/bin/jsii-pacmak --code-only
  | Workdir: /var/folders/0d/vfk25q7d7l33v01fb1hy2vgw0000gp/T/temp-QAJUII
  +----------------------------------------------------------------------------------
    at newError (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-srcmak/lib/util.js:38:20)
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-srcmak/lib/util.js:55:29)
    at Object.onceWrapper (events.js:418:26)
    at ChildProcess.emit (events.js:311:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
⠏ downloading and generating modules and providers...
jsii compilation failed with non-zero exit code: 1
  | Error: Cannot find module 'json-stable-stringify/package.json'
  | Require stack:
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/type-system.js
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/index.js
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/lib/index.js
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/bin/jsii-pacmak.js
  | - /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/bin/jsii-pacmak
  |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
  |     at Function.resolve (internal/modules/cjs/helpers.js:83:19)
  |     at TypeSystem._loadModule (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/type-system.js:105:40)
  |     at async TypeSystem._loadModule (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/type-system.js:109:17)
  |     at async TypeSystem.loadModule (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-reflect/lib/type-system.js:68:21)
  |     at async /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/lib/index.js:45:13
  |     at async Promise.all (index 0)
  |     at async Object.pacmak (/usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/lib/index.js:41:5)
  +----------------------------------------------------------------------------------
  | Command: /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii-pacmak/bin/jsii-pacmak --code-only
  | Workdir: /var/folders/0d/vfk25q7d7l33v01fb1hy2vgw0000gp/T/temp-QAJUII
  +----------------------------------------------------------------------------------```
```
### Steps to Reproduce

-  ```cdktf init --template="python" --local```
- Update the cdktf.json as below 
```
{ 
"language": "python", 
"app": "poetry run python main.py", 
"terraformProviders": ["aws@~>3.40.0", "oktadeveloper/okta@~>3.10.1"],
  "terraformModules": [],
  "codeMakerOutput": "imports",
  "context": {
    "excludeStackIdFromLogicalIds": "true",
    "allowSepCharsInLogicalIds": "true"
  }
} 
```
- Run ``` cdktf get -l python ```

### Important Factoids

- Fails in MacOSX Catalina 10.15.7 (19H524)
- Works in CentOS8

### References

* #0000

**Comment by danieldreier** [comment]
Thank you for reporting this!

I've tried to reproduce this using the `cdktf.json` provided, Python 3.9.5, node 16.1.0, and cdktf 0.3.0, and OS X Catalina 10.15.7 (on X86) and it worked as expected. I'll go and try to reproduce using the exact same python and node versions as you but so far I cannot trigger this on my side.

Can you clarify whether this is on M1 or X86?

**Comment by karthikns16** [comment]
Hi @danieldreier , it is an X86
![image](https://user-images.githubusercontent.com/12508473/118902569-c1d81300-b8ca-11eb-9a49-dad48f259aba.png)

**Comment by skorfmann** [comment]
In general this looks like a broken dependency installation with jsii. However, since this failed package doesn't seem to be actually necessary, #715 drops the dependency.

**Comment by karthikns16** [comment]
@skorfmann  will there be a 0.3.1 version available soon with this fix?

**Comment by skorfmann** [comment]
There's no immediate plan, but there's likely a new release soonish. You can use the pre-release version in the meantime though https://www.npmjs.com/package/cdktf/v/0.3.0-pre.239

**Comment by skorfmann** [comment]
@karthikns16 were you able to fix it?

**Comment by karthikns16** [comment]
@skorfmann 

It worked 

```
cdktf get
Newer version of Terraform CDK is available [0.3.0] - Upgrade recommended
Generated python constructs in the output directory: imports
```

**Comment by skorfmann** [comment]
Cool, thanks for the feedback! Closing this then.
